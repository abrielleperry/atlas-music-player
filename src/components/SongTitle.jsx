import React from "react";

export default function SongTitle() {
  return (
    <div className="flex flex-col mb-3 mt-6">
      <p className="font-bold text-2xl dark:text-white">Tidal Drift</p>
      <p className="text-base text-natural-stone dark:text-zinc-200">
        Echos of the Sea
      </p>
    </div>
  );
}
