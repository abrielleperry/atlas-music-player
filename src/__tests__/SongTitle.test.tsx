import { expect, test } from "vitest";
import { render } from "@testing-library/react";
import SongTitle from "../components/SongTitle";

test("SongTitle renders with title and artist", () => {
  const { asFragment } = render(
    <SongTitle title="Painted in Blue" artist="Soul Canvas" />
  );
  expect(asFragment()).toMatchSnapshot();
});

test("SongTitle renders when title and artist are empty", () => {
  const { asFragment } = render(<SongTitle title="" artist="" />);
  expect(asFragment()).toMatchSnapshot();
});
