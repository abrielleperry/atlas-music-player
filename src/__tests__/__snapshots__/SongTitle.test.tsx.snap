// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SongTitle renders when title and artist are empty 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col mb-3 mt-6"
  >
    <p
      class="font-bold text-2xl dark:text-white"
    />
    <p
      class="text-base text-natural-stone dark:text-zinc-200"
    />
  </div>
</DocumentFragment>
`;

exports[`SongTitle renders with title and artist 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col mb-3 mt-6"
  >
    <p
      class="font-bold text-2xl dark:text-white"
    >
      Painted in Blue
    </p>
    <p
      class="text-base text-natural-stone dark:text-zinc-200"
    >
      Soul Canvas
    </p>
  </div>
</DocumentFragment>
`;
